services/bettersocial/post/blockAnonymousPost.js

```diff
@@ -54,8 +53,8 @@ const BetterSocialBlockAnonymousPost = async (token, selfUserId, postId, source,
       await UserFollowUserFunction.userBlock(
         UserFollowUser,
         UserFollowUserHistory,
-        selfUserId,
         authorAnonymousUserId,
+        selfUserId,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of arguments in `UserFollowUserFunction.userBlock` has been changed. Ensure that this change is reflected wherever this function is called to avoid incorrect behavior.
```diff
-    authorAnonymousUserId,
-    selfUserId,
+    selfUserId,
+    authorAnonymousUserId,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->